src/Symfony/Bundle/FrameworkBundle/Command/RouterMatchCommand.php

```diff
@@ -75,6 +76,8 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        $output = new SymfonyStyle($input, $output);
```

Comment:
Overriding a method argument imho is no good style (see also https://github.com/symfony/symfony/pull/14595#discussion_r30175159 for the same).